src/defaults/components.js

```diff
@@ -68,7 +68,7 @@ const defaults = {
       horizontal: 'shopify-buy__layout-horizontal',
     },
     text: {
-      button: 'SHOP NOW',
+      button: 'ADD TO CART',
```

Comment:
Agree with the text change, but why is this in all caps? Should it be the default?